autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/AutofillJavascriptInterface.kt

```diff
@@ -175,10 +173,17 @@ class AutofillStoredBackJavascriptInterface @Inject constructor(
     fun storeFormData(data: String) {
         Timber.i("storeFormData called, credentials provided to be persisted")
 
-        getAutofillDataJob += coroutineScope.launch(dispatcherProvider.default()) {
+        getAutofillDataJob += coroutineScope.launch(dispatcherProvider.default() + handler) {
+
             val currentUrl = currentUrlProvider.currentUrl(webView) ?: return@launch
 
             val request = requestParser.parseStoreFormDataRequest(data).credentials
+
+            if (request.username == null && request.password == null) {
```

Comment:
NIT: Edge case - save dialog is shown when you put space only on both username and password. Looks a bit weird in edit mode.